resources/views/layouts/datagrid/_togglers.blade.php

```diff
@@ -7,7 +7,7 @@
         <div class="dropdown-menu hidden" role="menu" id="toggler-submenu">
             @foreach ($model->datagridSortableColumns() as $field => $translation)
                 @php
-                    $options = [$campaign, 'order' => $field];
+                    $options = [$campaign, 'order' => $field, 'bookmark' => request()->get('bookmark')];
```

Comment:
blade should never ever call request(), the controller is responsible for giving parameters